1-js/06-advanced-functions/03-closure/10-make-army/solution.md

```diff
@@ -51,24 +51,29 @@ function makeArmy() {
 }
 ```
 
-...We can see that it lives in the lexical environment associated with the current `makeArmy()` run. But when `army[5]()` is called, `makeArmy` has already finished its job, and `i` has the last value: `10` (the end of `while`).
+ we can see that it lives in the lexical environment associated with the current `makeArmy()` run. But when `army[5]()` is called, `makeArmy` has already finished its job, and the final value of `i` is `10` (at the end of `while`).
```

Comment:
we → We?

Reply:
I was a bit confused about that, but then I went with small letter because it is a continuation of previous sentence. I can change it if you want because either is fine. Just, the first one made more sense to me!

Reply:
Okay. I will change it.